### PR TITLE
Add linkerd chart

### PR DESCRIPTION
**Pull request by viglesiasce**
https://linkerd.io/

This was generated from `helm create` so most of the things we find in review will need to be updated there too.

### CODE REVIEW

File: stable/linkerd/templates/NOTES.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  echo http://$NODE_IP:$NODE_PORT/login
+{{- else if contains "LoadBalancer" .Values.service.type }}
+     NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+           You can watch the status of by running 'kubectl get svc -w {{ template "fullname" . }}'
+  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+  echo http://$SERVICE_IP:9990
+{{- else if contains "ClusterIP"  .Values.service.type }}
+  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "fullname" . }}" -o jsonpath="{.items[0].metadata.name}")
+  echo "Visit http://127.0.0.1:9990 to use your application"
+  kubectl port-forward $POD_NAME 9990:9990 4140:4140
```
Comment (lachie83) [inline stable/linkerd/templates/NOTES.txt]: Can you please add `--namespace= {{ .Release.Namespace }}`

File: stable/linkerd/templates/service.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  ports:
+  - name: outgoing
+    port: 4140
+  - name: incoming
+    port: 4141
+  - name: admin
+    port: 9990
+  selector:
+    app: {{ template "fullname" . }}
+  type: LoadBalancer
```
Comment (lachie83) [inline stable/linkerd/templates/service.yaml]: Is this needed given the type on line 13?

File: stable/linkerd/values.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Default values for linkerd.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+image:
+  linkerd:
+    repository: buoyantio/linkerd
+    tag: 0.8.2
```
Comment (lachie83) [inline stable/linkerd/values.yaml]: Should we bump to 0.8.3 now or in another PR?

File: stable/linkerd/templates/NOTES.txt [hunk 4]
```diff
@@ -0,0 +1,18 @@
+Add ports on your services that have the name {{ .Values.linkerd.portName }}
```
Comment (lachie83) [inline stable/linkerd/templates/NOTES.txt]: By default this statement renders as

> Add ports on your services that have the name service

I feel that's a little vague given no prior knowledge to linkerd. Can we provide an example and make that `service` pop a little?

**Review by lachie83 — CHANGES_REQUESTED** [review]
Couple of small UX related changes

**Comment by lachie83** [comment]
Please add README

**Comment by viglesiasce** [comment]
I believe I have things patched up now.

**Comment by viglesiasce** [comment]
@k8s-bot e2e test this

File: stable/linkerd/Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: Service mesh for cloud native apps
+name: linkerd
```
Comment (prydonius) [inline stable/linkerd/Chart.yaml:3]: Can you add the appropriate metadata for home, source, maintainers, icon and some keywords? e.g.

```
home: https://linkerd.io/
sources:
- https://github.com/BuoyantIO/linkerd
maintainers:
- name: Vic Iglesias
  email: viglesias@google.com
```

File: stable/linkerd/templates/config.yaml [hunk 6]
```diff
@@ -0,0 +1,28 @@
+---
```
Comment (prydonius) [inline stable/linkerd/templates/config.yaml]: These separators are not generally need, as Helm automatically places them.

File: stable/linkerd/templates/config.yaml [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}-config
```
Comment (prydonius) [inline stable/linkerd/templates/config.yaml:4]: Should this have an app label?

**Review by prydonius — CHANGES_REQUESTED** [review]
Sorry for the delay in looking at this @viglesiasce. Just a couple of small comments.

**Comment by viglesiasce** [comment]
All patched up folks. Thanks for the feedback. Will roll this all up to a PR to the `helm create` command.

**Comment by prydonius** [comment]
Worked great for me @viglesiasce, thanks!

**Review by prydonius — APPROVED** [review]

**Comment by prydonius** [comment]
@lachie83 since you've requested changes, I'll give you a chance to take a quick look

**Review by lachie83 — APPROVED** [review]

**Comment by lachie83** [comment]
LGTM

**Pull request merged**